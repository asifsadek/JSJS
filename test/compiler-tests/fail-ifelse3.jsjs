val a = if 1 == 1 
        then {
            print(7);
            7;
        }else{
            "this should fail";
        };

