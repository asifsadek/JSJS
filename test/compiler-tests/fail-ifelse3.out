Type error: expected value of type 'string', got a value of type 'num' instead
